wqflask/tests/wqflask/show_trait/testSampleList.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+import re
+from unittest import mock
+from wqflask.show_trait.SampleList import natural_sort
+
+
+class TestSampleList(unittest.TestCase):
+    def test_natural_sort(self):
+        "Sort the list into natural alphanumeric order."
+
+        characters_list = ["z", "f", "q", "s", "t", "a", "g"]
+        names_list = ["temp1", "publish", "Sample", "Dataset"]
+
+        natural_sort(characters_list)
+        natural_sort(names_list)
```

Comment:
**suggestion:** Good point for refactoring. Notice how `natural_sort` is side-effectful i.e. modifies the list. We should probably refactor this to:
- if we are still passing the list as a param, we should copy it, modify that copy when sorting and return it.
- Have a return value
cc @zsloan

Reply:
Great idea, just did that in this PR.